js/vnext/projection/cells.js

```diff
@@ -12,10 +12,19 @@ function translateRow(columnGroup, row) {
     };
   }
   if (_.has(row, 'item')) {
-    return {
+    const obj =  {
       classes: row.classes,
-      cells: _.map(columnGroup.leafColumns, col => (row.item[col.name] || {})),
+      cells: _.map(columnGroup.leafColumns, col => {
+        const cell = row.item[col.name] || {};
+        const tpl = col.template || col.headTemplate || col.footTemplate;
+        if (tpl) {
+          cell.html =  _.isFunction(tpl) ? tpl(row.item) : tpl;
+        }
+        return cell;
```

Comment:
You can not return the `row.item[col.name]` directly, it must be a separated copy